tests/test_sync_synonym.py

```diff
@@ -0,0 +1,101 @@
+"""Synonym synchronization tests."""
+import os
+import sys
+import unittest
+from pathlib import Path
+from typing import Dict, Union, List
+
+import pandas as pd
+
+# noinspection DuplicatedCode
+TEST_DIR = Path(os.path.abspath(os.path.dirname(__file__)))
+PROJECT_ROOT = TEST_DIR.parent
+sys.path.insert(0, str(PROJECT_ROOT))
+from src.scripts.sync_synonym import sync_synonyms
+
+ONTO_DIR = PROJECT_ROOT / 'src' / 'ontology'
+TMP_DIR = ONTO_DIR / 'tmp'
+META_DIR = ONTO_DIR / 'metadata'
+IN_DIR = TEST_DIR / 'input'
+OUT_DIR = TEST_DIR / 'output'
+INPUT_MONDO_DB = IN_DIR / 'mondo.db'  # todo: do i need this var? or just refer by relative pathing when creating?
+INPUT_SOURCE_DB = IN_DIR / 'omim.db'
+INPUT_MONDO_SYNONYMS = IN_DIR / 'mondo-synonyms-scope-type-xref.tsv'
+INPUT_MAPPINGS = TMP_DIR / 'mondo.sssom.tsv'
+INPUT_SOURCE_METADATA = META_DIR / 'omim.yml'
+OUTPUT_ADDED = OUT_DIR / 'omim.synonyms.added.robot.tsv'
+OUTPUT_CONFIRMED = OUT_DIR / 'omim.synonyms.confirmed.robot.tsv'
+OUTPUT_DELETED = OUT_DIR / 'omim.synonyms.deleted.robot.tsv'
+OUTPUT_UPDATED = OUT_DIR / 'omim.synonyms.updated.robot.tsv'
+
+# TODO: Change names e.g. mondo-test.owl, mondo-test-syn-sync.owl
+class TestSyncSynonyms(unittest.TestCase):
+    """Synonym synchronization tests."""
```

Comment:
## Tests

Reply:
I will leave it up to Trish to involve me in this if she feels it helps with something!